### DISCUSSION TITLE
[BUG]: updateListingInventory with country-specific prices and product variations produces wrong prices

### DISCUSSION THREAD

**Discussion by cajudosu** [etsy/open-api#707] [Q&A]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Describe the bug

If you have a product, that has the switch for "Domestic & global pricing" activated, the product has two variations for which you track the quantities seperately and you perform a quantitey update via updateListingInventory, the decimal seperator for the main country changes from comma (",", which is the correct seperator here in Germany) to point ("."). As a result, a price like 39,95 becomes 39.95. If you oversee this and save the product the price becomes 3995,00 (100 times as much!)

### Steps to reproduce

1. Add an article in Etsy
2. Under "Inventory and pricing" activate "Domestic & global pricing"
3. For the domestic country set a price e.g. of 1,00€ and for Everywhere else set a price of 2,00€
4. Add two variations e.g. on the basis of the primary color e.g. "Beige" and "Black" and activate seperate Tracking of the quantities for each variation
5. Set the quantities to 10 for each variation
6. Save the product (see screenshots for example)
7. Update quantities via Etsy API v3 endpoint updateListingInventory with a payload as shown in Appendix 1
8. Take a look at "Inventory and pricing" in the Shop Manager, the price has changed from 1,00 to 1.00.

```json
PUT https://openapi.etsy.com/v3/application/listings/849932477/inventory HTTP/1.1
Authorization: Bearer 15329666.qllRhl_V-45twsIqmUSlZDugf5G6h4s3enOvsQKkw4k74QTWjzkcY3THETdAQGJGSAnp_Y7OrehjnuIkL9s5K5Jr1aq
x-api-key: 3tjpw4q3bgymsp7n3ach7mdk
Content-Type: application/json
Host: openapi.etsy.com
Content-Length: 497

{
    "products": [
        {
            "sku": "",
            "property_values": [
                {
                    "property_id": 200,
                    "value_ids": [
                        106359401894
                    ],
                    "scale_id": null,
                    "property_name": "Primary color",
                    "values": [
                        "Beige"
                    ]
                }
            ],
            "offerings": [
                {
                    "quantity": 8,
                    "price": 2.0,
                    "is_enabled": true
                }
            ]
        },
        {
            "sku": "",
            "property_values": [
                {
                    "property_id": 200,
                    "value_ids": [
                        83603659447
                    ],
                    "scale_id": null,
                    "property_name": "Primary color",
                    "values": [
                        "Schwarz"
                    ]
                }
            ],
            "offerings": [
                {
                    "quantity": 10,
                    "price": 2.0,
                    "is_enabled": true
                }
            ]
        }
    ],
    "price_on_property": [],
    "quantity_on_property": [
        200
    ],
    "sku_on_property": []
}
```
![Gartenhandschuhe-Artikel-bearbeiten-Etsy__before](https://user-images.githubusercontent.com/14168155/197535174-dfb5e55c-4fb6-48fc-a32e-6694ba048adf.png)
![Gartenhandschuhe-Artikel-bearbeiten-Etsy__after](https://user-images.githubusercontent.com/14168155/197535201-5bbcbc77-935b-4744-a51d-5972080743d9.png)

### Expected behavior

After the stock update via updateListingInventory the price still has a comma (",") as decimal separator.

**Comment by etsyachristensen** [comment]
The OpenAPI documentation states that the price value for the inventory data when being submitted is a `float`. A floating point number, is a positive or negative whole number with a decimal point. There are no commas in floating point numbers, regardless of location. Unfortunately, we won't be able to make a change to this endpoint to accomodate regionally formatted numbers, only floats.

**Comment by cajudosu** [comment]
As you can see in the payload above we correctly transfer the price as a JSON float value i.e. `2.0` with a point as decimal separator. That is not the point. The point is, that assuming the product is configured as stated above, something goes wrong in Etsy displaying the price in the wrong format causing wrong prices upon saving the product the next time.

**Comment by etsyachristensen** [comment]
@cajudosu I think I understand. I will raise this with our international teams.

**Comment by cajudosu** [comment]
Thanks a lot 😃

**Comment by etsyachristensen** [comment]
@cajudosu we will be disabling the Issues board, so we are converting this to a discussion topic. We are still tracking this as an active ticket in our system.

**Comment by DanielAtBillbee** [comment]
@etsyachristensen Any update on this?

**Reply by Gareth064** [reply]
etsyachristensen doesn't work for Etsy anymore.

You will need to email developer@etsy.com to ask for an update on this
